### PR TITLE
[branch/v6.0] Update getting started for cloud

### PR DESCRIPTION
**Pull request by stevenGravy**
- show verify signature for client tool install
- show how to add a new node and access via ssh
- correct dir for enterprise, `teleport-ent`, not `teleport`

### CODE REVIEW

File: docs/pages/cloud/getting-started.mdx [hunk 1]
```diff
@@ -29,23 +33,58 @@ Login into Teleport and test the connection:
 ```bash
 # tsh logs you in and receives short-lived certificates
 $ tsh login --proxy=myinstance.teleport.sh:443 --user=email@example.com
-# tctl is a cluster configuration tool
+# Your initial role is admin as the creater of the account
```
Comment (klizhentas) [inline docs/pages/cloud/getting-started.mdx:36]: Let's not advertise the admin role as a default role

File: docs/pages/cloud/getting-started.mdx [hunk 2]
```diff
@@ -29,23 +33,58 @@ Login into Teleport and test the connection:
...
 
+## Step 4/5 Install the Teleport agent on a Linux Server
 Generate a [dynamic token](../admin-guide.mdx#short-lived-dynamic-tokens):
 
 ```bash
-$ tctl nodes add --ttl=5m --roles=node --token=$(uuid)
+$ tctl nodes add --ttl=20m --roles=node --token=$(uuid)
+```
+
+Install Teleport on a Linux server
```
Comment (klizhentas) [inline docs/pages/cloud/getting-started.mdx:57]: What's the benefit of this compared to our wizard?

**Review by klizhentas — CHANGES_REQUESTED** [review]

File: docs/pages/cloud/getting-started.mdx (hunk 2 above)
Comment (stevenGravy) [reply docs/pages/cloud/getting-started.mdx:57]: @klizhentas  I can change to using the wizard.  Will take out the nodes add part then.  Will add screen shots then.

**Comment by stevenGravy** [comment]
@klizhentas  please take a look at the updates.  This shows the user how to use the Add Server and accessing from the CLI

File: docs/pages/cloud/getting-started.mdx [hunk 3]
```diff
@@ -3,20 +3,47 @@ title: Getting Started With Teleport Cloud
...
 Install client libraries:
 
 <Tabs>
   <TabItem label="Linux">
     ```bash
-    curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    cd teleport
+    curl -O https://get.gravitational.com/teleport-ent-v{{ teleport.version }}-linux-amd64-bin.tar.gz
+    # verify signature
```
Comment (klizhentas) [inline docs/pages/cloud/getting-started.mdx:43]: it's now a different format: https://goteleport.com/docs/docs/best-practices/#varibles

`(=teleport.version=)`

**Review by klizhentas — CHANGES_REQUESTED** [review]

File: docs/pages/cloud/getting-started.mdx [hunk 4]
```diff
@@ -3,20 +3,47 @@ title: Getting Started With Teleport Cloud
...
 
 <Tabs>
   <TabItem label="Linux">
     ```bash
     curl -O https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    tar -xzf teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz
-    cd teleport
+    # verify signature 
+    echo "$(curl  https://get.gravitational.com/teleport-ent-v(=teleport.version=)-linux-amd64-bin.tar.gz.sha256)" | sha256sum --check 
+    tar -xzf teleport-ent-v{{ teleport.version }}-linux-amd64-bin.tar.gz
```
Comment (klizhentas) [inline docs/pages/cloud/getting-started.mdx:45]: same here

**Review by klizhentas — APPROVED** [review]

**Pull request merged**